libs/vscode-ui/components/src/lib/input/input.component.html

```diff
@@ -1,3 +1,28 @@
-<p>
-  input works!
-</p>
+<!-- description -->
+<div class="setting-item-description" [id]="descriptionId">
+  {{ field.description }}
+</div>
+
+<div class="setting-item-modified-indicator"></div>
+
+<div class="setting-item-value">
+  <div class="setting-item-control">
+    <div class="monaco-inputbox">
+      <input
+        (keyup)="updateValue($event.target.value)"
+        [value]="value"
+        class="input setting-control-focus-target"
+        type="text"
+        role="textbox"
+        [required]="field.required"
+        [attr.list]="field.enum ? field.name + '_enum' : null"
```

Comment:
very small nit: I think undefined is the more HTML standard approved way of restoring properties to their default state.